Scripts/Build-Phases/sparkle-code-sign.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+
+set -e
+
+# Codesign
+#
+IDENTITY="Developer ID Application: Automattic, Inc."
+SPARKLE_BIN="${SRCROOT}/External/Sparkle/bin"
+CODESIGN_TOOL="${SPARKLE_BIN}/codesign_embedded_executable"
+CODESIGN_HARDENEDRUNTIME_CMD="codesign --force -o runtime -s"
+
+# XPC
+#
+XPC_SOURCE="${SRCROOT}/External/Sparkle/XPCServices"
+XPC_TARGET="${BUILT_PRODUCTS_DIR}/${FRAMEWORKS_FOLDER_PATH}/../XPCServices"
+
+# Services
+#
+XPC_NETWORK="org.sparkle-project.InstallerConnection.xpc"
+XPC_LAUNCHER="org.sparkle-project.InstallerLauncher.xpc"
+XPC_STATUS="org.sparkle-project.InstallerStatus.xpc"
+
+# UPDATER
+UPDATER_TARGET="${BUILT_PRODUCTS_DIR}/${FRAMEWORKS_FOLDER_PATH}/Sparkle.framework/Versions/A/Resources"
+SPARKLE_AUTOUPDATE="Autoupdate"
+SPARKLE_UPDATER="Updater.app/Contents/MacOS/Updater"
+
+# Bail out whenever we're not in Release
+#
+if [ "${CONFIGURATION}" != "Release" ]; then
+    echo "NOT Signing in debug!"
+    exit 0
+fi
+
+# Reset: Sparkle's Codesign fails when attempting to re-sign
+#
+echo "Nuking old XPC..."
+
+rm -r "${XPC_TARGET}"
```

Comment:
```suggestion
rm -rf "${XPC_TARGET}"
```

This will fail on a clean build directory because the directory doesn't exist. Using `-rf` fixes that.

Reply:
Can you believe that I actually had a commit that used `-rf` then I removed it because I thought the failure I experienced when I added it was due to version `1.x` not having the XPC stuff?

🤦